src/test/java/io/kestra/plugin/microsoft365/sharepoint/ExportTest.java

```diff
@@ -0,0 +1,291 @@
+        DriveItemRequestBuilder mockDriveItems = mock(DriveItemRequestBuilder.class);
+        DriveItemItemRequestBuilder mockDriveItem = mock(DriveItemItemRequestBuilder.class);
+        ContentRequestBuilder mockContent = mock(ContentRequestBuilder.class);
+
+        when(mockClient.drives()).thenReturn(mockDrives);
+        when(mockDrives.byDriveId(anyString())).thenReturn(mockDriveItems);
+        when(mockDriveItems.items()).thenReturn(mock(com.microsoft.graph.drives.item.items.ItemsRequestBuilder.class));
+        when(mockDriveItems.items().byDriveItemId(anyString())).thenReturn(mockDriveItem);
+        when(mockDriveItem.content()).thenReturn(mockContent);
+
+        // Mock get() for file metadata
+        DriveItem mockDriveItemResponse = new DriveItem();
+        mockDriveItemResponse.setId("word-doc-id");
+        mockDriveItemResponse.setName("document.docx");
+        mockDriveItemResponse.setSize(10240L);
+        mockDriveItemResponse.setWebUrl("https://contoso.sharepoint.com/document.docx");
+
+        when(mockDriveItem.get()).thenReturn(mockDriveItemResponse);
+
+        // Mock content download with format conversion
+        String pdfContent = "Mock PDF content";
+        InputStream mockInputStream = new ByteArrayInputStream(pdfContent.getBytes());
+        when(mockContent.get(any())).thenReturn(mockInputStream);
+
+        // Create a spy of the task to override connection method
+        Export testTask = spy(exportTask);
+        doReturn(mockConnection).when(testTask).connection(any(RunContext.class));
+
+        // When
+        Export.Output output = testTask.run(runContext);
+
+        // Then
+        assertThat(output.getItemId(), is("word-doc-id"));
+        assertThat(output.getOriginalName(), is("document.docx"));
+        assertThat(output.getName(), is("document.pdf"));
+        assertThat(output.getUri(), notNullValue());
+        assertThat(output.getFormat(), is("pdf"));
+        assertThat(output.getWebUrl(), is("https://contoso.sharepoint.com/document.docx"));
+
+        // Verify the format parameter was used
+        verify(mockContent).get(any());
+    }
+
+    @Test
+    void shouldExportExcelToPdf() throws Exception {
+        // Given
+        RunContext runContext = runContextFactory.of();
+        SharepointConnection mockConnection = mock(SharepointConnection.class);
+        GraphServiceClient mockClient = mock(GraphServiceClient.class);
+
+        Export exportTask = Export.builder()
+            .tenantId(Property.ofValue("test-tenant-id"))
+            .clientId(Property.ofValue("test-client-id"))
+            .clientSecret(Property.ofValue("test-client-secret"))
+            .siteId(Property.ofValue("test-site-id"))
+            .driveId(Property.ofValue("test-drive-id"))
+            .itemPath(Property.ofValue("/Documents/spreadsheet.xlsx"))
+            .format(Property.ofValue(Export.FormatType.valueOf("PDF")))
```

Comment:
Hmm directly use the value everywhere no?

```suggestion
            .format(Property.ofValue(Export.FormatType.PDF))
```

Reply:
yes yes let me do this everywhere